unit_tests/test_version_utils.py

```diff
@@ -370,7 +370,7 @@ def test_scylla_versions_decorator_negative_latest_scylla_no_attr():
 @pytest.mark.integration
 @pytest.mark.need_network
 @pytest.mark.skip(reason="those are integration tests only")
-@pytest.mark.parametrize('docker_repo', ['scylladb/scylla-nightly', 'scylladb/scylla', 'scylladb/scylla-enterprise'])
```

Comment:
After keeping the old logic for the stable repo we should removechanges to this file.